packages/common/src/discussions/edit.ts

```diff
@@ -114,6 +153,9 @@ export async function deleteDiscussion(
   id: string,
   requestOptions: IUserRequestOptions
 ): Promise<void> {
+  removeSetting({ id }).catch(() => {
```

Comment:
Better practice to only delete the settings if the item was successfully deleted. `requestOptions` needs updated to `IHubRequestOptions` and spread onto object passed to removeSetting for auth to work on the settings API request.

```jsx
export async function deleteDiscussion(
  id: string,
  requestOptions: IHubRequestOptions
): Promise<void> {
  const ro = { ...requestOptions, ...{ id } } as IUserItemOptions;
  await removeItem(ro);
  try {
    removeSetting({ id, ...requestOptions });
  } catch (e) {
    // suppress error
  }
  return;
}
```